include/wx/gtk/mimetype.h

```diff
@@ -23,7 +23,11 @@
 class WXDLLIMPEXP_CORE wxGTKMimeTypesManagerImpl : public wxMimeTypesManagerImpl
 {
 protected:
+#if defined(__UNIX__)
     wxString GetIconFromMimeType(const wxString& mime) wxOVERRIDE;
+#else
+    wxString GetIconFromMimeType(const wxString& mime);
```

Comment:
Sorry, why does it fail to work? AFAICS the only code using this function (`GetIconFromMimeType()`) is in Unix code, so either it should be declared in the base class (and then `override` can be used) or it shouldn't be used at all.

So what exactly goes wrong if you omit the `#else` branch entirely?